package.json

```diff
@@ -6,7 +6,9 @@
     "node-hc": "bin/appmetrics-cli.js"
   },
   "dependencies": {
+    "heapdump": "0.x",
     "nan": "2.x",
+    "nodereport": "1.x",
```

Comment:
It's binding.gyp that gets replaced entirely, package.json just has one part replaced inline ("node-gyp rebuild" to "node extract_all_binaries.js") so I think we'll be ok with this change.